common/scala/src/main/scala/org/apache/openwhisk/core/entity/ExecManifest.scala

```diff
@@ -58,6 +58,20 @@ protected[core] object ExecManifest {
     mf
   }
 
+  /**
+   * Reads runtimes manifest from runtime string
+   *
+   * @param runtime
+   * @return the manifest if initialized successfully, or an failure
+   */
+  protected[core] def initialize(runtime: String): Try[Runtimes] = {
+    val rmc = loadConfigOrThrow[RuntimeManifestConfig](ConfigKeys.runtimes)
+    val mf = Try(runtime.parseJson.asJsObject).flatMap(runtimes(_, rmc))
+    var manifest: Option[Runtimes] = None
```

Comment:
this is incorrect - the reason we do the assignment to `manifest` on line 57 is to set the singleton. Here, this has no effect because the variable is local.

Reply:
I already changed to use existed initialize method.